apps/SageAccountWeb/src/components/CertificationQuiz.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { Container } from 'react-bootstrap'
 import { Link } from 'react-router-dom'
 import { CertificationQuiz as Quiz } from 'synapse-react-client'
-import ReturnArrow from '../assets/ReturnArrow.svg'
+import ReturnArrow from '../assets/ReturnArrow.svg?url'
```

Comment:
Upgrading `vite-plugin-svgr` resulted in a breaking change -- the end result is that we cannot import SVGs as strings using the default export AND as React components using `import { ReactComponent as ...} from ...`.

However, Vite natively (no plugin required!) handles importing resource URLs if we add `?url`. The plugin was configured to still support using `import { ReactComponent as ... } from ...`.